### PR TITLE
Fix #31947 Added version parameter to all JS files breaks MathJax in the Browser

### PR DESCRIPTION
**Pull request by fneumann**
https://mantis.ilias.de/view.php?id=31947

### CODE REVIEW

**Comment by chfsx** [comment]
Thanks @fneumann!

File: src/GlobalScreen/Scope/Layout/MetaContent/Media/AbstractMediaWithPath.php [hunk 1]
```diff
@@ -13,7 +13,12 @@ public function getContent() : string
         // the version string is only appended if the content string is not
         // a data uri, otherwise the data uri will behave incorrectly.
         if (!$this->isContentDataUri($content)) {
-            return rtrim($content, "?") . "?" . $this->version;
+            if ($this->hasContentParameters($content)) {
+                return rtrim($content, "&") . "&" . $this->version;
+            }
+            else {
```
Comment (mjansenDatabay) [inline src/GlobalScreen/Scope/Layout/MetaContent/Media/AbstractMediaWithPath.php:19]: @fneumann / @chfsx This violates `PSR 2 + X` ;-)

File: src/GlobalScreen/Scope/Layout/MetaContent/Media/AbstractMediaWithPath.php [hunk 2]
```diff
@@ -27,4 +32,8 @@ protected function isContentDataUri(string $content) : bool
         return (bool) preg_match('/^(data:)([a-z\/]*)((;base64)?)(,?)([A-z0-9=]*)$/', $content);
     }
 
+    protected function hasContentParameters(string $content): bool
```
Comment (mjansenDatabay) [inline src/GlobalScreen/Scope/Layout/MetaContent/Media/AbstractMediaWithPath.php:35]: This also violates `PSR-2 + X`

Comment (chfsx) [reply src/GlobalScreen/Scope/Layout/MetaContent/Media/AbstractMediaWithPath.php:35]: This `+ X` was a very expensive decision ;-) I'll fix that

Comment (mjansenDatabay) [reply src/GlobalScreen/Scope/Layout/MetaContent/Media/AbstractMediaWithPath.php:35]: I agree :-).

**Pull request merged**